officemanager/department/tests/tests.py

```diff
@@ -1,3 +1,98 @@
 from django.test import TestCase
+from rest_framework.test import APITestCase
+from department.models import Department, Employee
 
-# Create your tests here.
+class StringReprTestCase(TestCase):
```

Comment:
The following unit tests have been missed:
1. Get department by id
2. Get employee by id
3. In your tests, you consider only positive cases. What about negative cases (ex. request for a non-existent department or employee, put or post with missing required parameters - corresponding HTTP codes should be returned and body response should include corresponding error message)?
4. It is necessary to check not only status codes, but also the response body. Status 200 doesn't guarantee that response body includes valid / correct data.